variables.tf

```diff
@@ -5,13 +5,13 @@ variable "create_bucket" {
 }
 
 variable "attach_elb_log_delivery_policy" {
-  description = "Controls if S3 bucket should have ELB log delivery policy attached"
```

Comment:
why are there all of these changes? ELB is correct - ALB and NLB are in fact ELB v2 (classic load balancer was ELB v1)